### PR TITLE
fix(docs): modify broken reflection example

### PR DESCRIPTION
**Pull request by lionelhorn**
### Summary of changes
Fixed code and imports of example

### Relinquishment of Rights

Please mark following checkbox to confirm that you relinquish all rights of your changes:

- [x] I waive and relinquish all rights regarding this changes (including code, text, and images) to Deepkit UG (limited), Germany. This changes (including code, text, and images) are under MIT license without name attribution, copyright notice, and permission notice requirement.

### CODE REVIEW

File: website/src/pages/documentation/runtime-types/getting-started.md [hunk 1]
```diff
@@ -33,20 +33,20 @@ Write your first code with runtime type information:
 _File: app.ts_
 
 ```typescript
-import { cast, MinLength, ReflectionClass } from '@deepkit/type';
+import { cast, MinLength, ReflectionClass, typeOf } from '@deepkit/type';
```
Comment (marcus-sa) [inline website/src/pages/documentation/runtime-types/getting-started.md:36]:
```suggestion
import { cast, MinLength, ReflectionClass } from '@deepkit/type';
```

**Review by marcus-sa — CHANGES_REQUESTED** [review]

File: website/src/pages/documentation/runtime-types/getting-started.md [hunk 2]
```diff
@@ -33,20 +33,20 @@ Write your first code with runtime type information:
...
 
-const user = cast<User>(User, {
+const user = cast<User>({
     username: 'Peter',
     birthDate: '2010-10-10T00:00:00Z'
 });
 console.log(user);
 
-const reflection = ReflectionClass.from(User);
+const reflection = ReflectionClass.from(typeOf<User>());
```
[the remark this answers is not in the document]
Comment (lionelhorn) [reply website/src/pages/documentation/runtime-types/getting-started.md]: Fixed in latest commit.

File: website/src/pages/documentation/runtime-types/getting-started.md (hunk 1 above)
Comment (lionelhorn) [reply website/src/pages/documentation/runtime-types/getting-started.md]: Fixed in latest commit.

**Comment by lionelhorn** [comment]
Closing this in favor of https://github.com/deepkit/deepkit-framework/pull/599

(Couldn't find a way to pass commitlint test, as past commits had a not accepted scope)

**Pull request closed without merging**